src/applications/claims-status/components/StemDeniedDetails.jsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import moment from 'moment';
+
+const StemDeniedDetails = ({
+  deniedAt,
+  isEnrolledStem,
+  isPursuingTeachingCert,
+}) => {
+  const date = moment(deniedAt).format('MMM D, YYYY');
+
+  return (
+    <>
+      <h1>Your Edith Nourse Rogers STEM Scholarship Application</h1>
+      <div className="vads-u-background-color--primary-alt-lightest vads-u-padding--2 vads-u-margin-bottom--3">
+        <h3 className="claims-alert-header vads-u-font-size--h4">
```

Comment:
I think this might be technically okay, since the h3 is in a div, but you may want to consider just making this a P tag, or an h2.

Reply:
I went ahead and changed this to be an h2 since I was already using the `vads-u-font-size--h4` class name here for styling.